<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QPushButton" name="countButton">
      <property name="text">
       <string>Count</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="heightLabel">
      <property name="text">
       <string>Height:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="cmLabel">
      <property name="text">
       <string>cm</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="weightLineEdit"/>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="resultLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="1" colspan="2">
     <widget class="QTextBrowser" name="infoTextBrowser"/>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="kgLabel">
      <property name="text">
       <string>kg</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="bmiLabel">
      <property name="text">
       <string>BMI:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="weightLabel">
      <property name="text">
       <string>Weight:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="heightLineEdit"/>
    </item>
    <item row="6" column="0">
     <widget class="QPushButton" name="closeButton">
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="infoLabel">
      <property name="text">
       <string>Info:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>503</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>251</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
